src/OrchardCore.Modules/OrchardCore.ArchiveLater/Indexes/ArchiveLaterPartIndex.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using YesSql.Indexes;
+
+namespace OrchardCore.ArchiveLater.Indexes;
+
+public class ArchiveLaterPartIndex : MapIndex
+{
+    public string ContentItemId { get; set; }
+
+    public bool Latest { get; set; }
+
+    public bool Published { get; set; }
+
+    public DateTime? ScheduledArchiveDateTimeUtc { get; set; }
```

Comment:
We are used to place the Ids at the beginning, okay, and Published and Latest at the very end

Reply:
I just order them by property name as I always did in any other app ;)

Reply:
Yes good to oder by names, particularly if there are many, otherwise I prefer to order by functionalities, e.g. `Color` and `HasColor` grouped together.

Here is to separate property levels, contentItem level properties as `ContentItemId` and e.g. `DisplayText` (when we use it) always at the beginning.

Then contentItem level states as `Published` and `Latest` always at the end as we do for other new index tables, but yes not always e.g. for the older index `ContentItemIndex` if I remember correctly.

Then other as part fields more related to a given index purpose in the middle.